varinfo/cf_config.py

```diff
@@ -103,33 +102,13 @@ def _read_config_file(self):
             for pattern in item['Variable_Pattern']
         }
 
-        self.global_supplements = {
```

Comment:
Just a general question: can we get global attributes by just doing `VarInfoBase.attrs` or something like that? For datasets that only have one group (the root group) it could be nice to have get the global attrs by default.

Reply:
I am on the fence about this:

- Pro: If we call it `VarInfoBase.global_attributes`, we are consistent with the previous publicly accessible class attributes on `VarInfoBase` and child classes. (Makes this less of a breaking change)
- Pro: The attribute name `global_attributes` is very intuitive and descriptive.
- Con: It feels a little contrary to the idea of cleaning up things to remove redundancy (although that redundancy is in the configuration file schema, which is _really_ confusing right now).
- Possible Con: This slightly reverts us to a place where global attributes are special. (Reason this might not be a con: Are global attributes indeed special? If so maybe this becomes a pro.)

Potential implementation:

```
class VarInfoBase(ABC):
    def __init__(...):
        ....
        self.global_attributes = self.groups['/']

    ...
```

@flamingbear - I'm curious what you think about this? (Tiebreaker!)

Reply:
I'm not super-inclined to add a special case here. this would just be the difference between

``` python
vi = VarInfoFromNetCDF4('somefile.nc)
vi.groups['/'].attributes
```

and
``` python
vi = VarInfoFromNetCDF4('somefile.nc)
vi.global_attributes
```
?

Ok, I'm a little torn if people have been using `vi.global_attributes` and are already used to it, I can see keeping it because it's not a huge change.
But it doesn't buy that many characters, if I'm understanding the question correctly.

Reply:
If it's only used by users and not in the code base I can see the convenience maybe

Reply:
So, honestly, I don't have a super strong opinion here.

I'm going to continue to leave `global_attributes` out in the interest of reducing what's in the class (given that the same information can be accessed via `VarInfoBase.groups`). But I would 100% add `VarInfoBase.global_attributes` as soon as someone says not having it has made things a pain for them.